verilog/preprocessor/verilog_preprocess_test.cc

```diff
@@ -234,6 +234,16 @@ TEST(VerilogPreprocessTest, TwoMacroDefinitions) {
   }
 }
 
+TEST(VerilogPreprocessTest, UndefMacro) {
```

Comment:
also suggest a test case that undef's a nonexistent macro.

Reply:
Done.